test/tools/libtesteth/TestOutputHelper.cpp

```diff
@@ -52,6 +52,16 @@ void TestOutputHelper::initTest(size_t _maxTests)
 
 bool TestOutputHelper::passTest(std::string const& _testName)
```

Comment:
The name of this function is mysterious.  How about calling it `setTestName`?

Reply:
this function returns fals if --singletest is set and testName does not match the specified singletest name. it also outputs the test name if it is being executed and remembers it to the variable for later debug messages.

Reply:
OK... `askToRunTest()`?

Reply:
checkTest() ?